### PR TITLE
Rft: DubboHeader

### PR DESCRIPTION
**Pull request by gaoxinge**
**Which issue(s) this PR fixes**:

Fixes https://github.com/dubbogo/hessian2/issues/46

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

**Comment by wongoo** [comment]
@gaoxinge it's a big change, we will review it carefully, possibly consider to merge it in a major release in the future.

**Comment by gaoxinge** [comment]
@wongoo Thanks!

File: hessian_test.go [hunk 1]
```diff
@@ -24,6 +24,8 @@ import (
 
 import (
 	"github.com/stretchr/testify/assert"
+	"log"
+	"os/exec"
```
Comment (AlexStocks) [inline hessian_test.go]: Pls use another import ()

Comment (gaoxinge) [reply hessian_test.go]: @AlexStocks Oh yes.

**Pull request closed without merging**